mmcv/transforms/wrappers.py

```diff
@@ -0,0 +1,307 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+from collections.abc import Sequence
+from typing import Any, Callable, Dict, List, Optional, Union
+
+import numpy as np
+
+import mmcv
+from .base import BaseTransform
+from .builder import TRANSFORMS
+from .utils import cache_random_params
+
+# Indicator for required but missing keys in results
+NotInResults = object()
+
+# Import nullcontext if python>=3.7, otherwise use a simple alternative
+# implementation.
+try:
+    from contextlib import nullcontext
+except ImportError:
+    from contextlib import contextmanager
+
+    @contextmanager
+    def nullcontext(resource=None):
+        try:
+            yield resource
+        finally:
+            pass
+
+
+@TRANSFORMS.register_module()
+class Compose(BaseTransform):
+    """Compose a data pipeline with a sequence of transforms.
+
+    Args:
+        transforms (list[dict | callable]): Either config
+          dicts of transforms or transform objects.
+    """
+
+    def __init__(self, transforms: List[Union[Dict, Callable[[Dict], Dict]]]):
+        assert isinstance(transforms, Sequence)
+        self.transforms = []
+        for transform in transforms:
+            if isinstance(transform, dict):
+                transform = TRANSFORMS.build(transform)
+                self.transforms.append(transform)
+            elif callable(transform):
```

Comment:
Should we use the `isinstance(transform)`?

Reply:
The built-in `callable()` serves the same purpose. We can also use `isinstance(transform, Callable)`.

Reply:
got it